src/Markup/Avalonia.Markup.Xaml.Loader/CompilerExtensions/Transformers/AvaloniaXamlIlWellKnownTypes.cs

```diff
@@ -101,6 +101,9 @@ class AvaloniaXamlIlWellKnownTypes
         public IXamlType IResourceDictionary { get; }
         public IXamlType ResourceDictionary { get; }
         public IXamlMethod ResourceDictionaryDeferredAdd { get; }
+        public IXamlType String { get; }
+        public IXamlType UriKind { get; }
+        public IXamlConstructor UriConstructor { get; }
```

Comment:
You don't need to create these types here, as you can access `context.Configuration.WellKnownTypes` to get `string` and maybe `UriKind`.